scrapy_autounit/utils.py

```diff
@@ -111,9 +111,29 @@ def parse_request(request, spider):
     parsed_request['headers'] = parse_headers(parsed_request['headers'])
     parsed_request['body'] = parsed_request['body'].decode('utf-8')
 
+    parsed_request['meta'] = {}
+    for key, value in parsed_request.get('meta').items():
```

Comment:
I think parsed_request.get('meta') will always be empty here, although I'm doubting myself.

Reply:
Ah, is the line above supposed to be `_meta = {}`?

Reply:
Ah yup, I guess I messed things there, thanks!